### ISSUE TITLE
AWS Clear Linux AMI Id

### ISSUE THREAD

**Issue by melardev** [clearlinux/clear-linux-documentation#1223]
**Do you think Clear Linux documentation needs a new document? Please describe.**
Please provide a clear and concise description of the title and content. Identify the target audience: Developer; System Administrator; or Basic User.
Target audience: Administrator
Hi, I am looking at how to automate the creation of AWS EC2 VMs with Clear linux, it is a little bit frustrating, such as basic thing is not documented as far as I can see, where can I find a reliable AMI id for use in scripts?

**Should the new document be a guide, a reference, or a tutorial?**
reference

**Comment by mvincerx** [comment]
Have you reviewed?: https://docs.01.org/clearlinux/latest/get-started/cloud-install/aws-web.html

**Comment by melardev** [comment]
@mvincerx Hi, thanks, and yes, I was there before asking, I have reviewed it very quickly now, still not able to find where I can clearly find the AMI for each region, that docs page only shows how to create a VM from the console, what I needed is the AMI id instead.

However, I found my response, we can find the AMI for each region under https://aws.amazon.com/marketplace/server/configuration?productId=4d505f99-1c49-45c1-b574-a59747ec19d6&ref_=psb_cfg_continue
but it does not seem to me it was documented. Am I wrong?

**Comment by lebensterben** [comment]
It's documented here:

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html

**Comment by melardev** [comment]
@lebensterben Can you specify more please? I was really interested in finding the SSM of clear linux, but did not find so I am using hardcoded ugly "ami-amiid" kind of syntax. That page does not explain how to get it for Clear linux, the only thing I see is running ec2 describe-images, which takes forever ...

**Comment by mvincerx** [comment]
https://aws.amazon.com/marketplace/pp/Intel-Clear-Linux-OS/B075TXF7KJ

**Comment by melardev** [comment]
@lebensterben I have just run describe-images command without any argument because I don't know how could I narrow it down, I got 238 MB of json output file ... yes, I got the AMI, but is that really needed to go through a 10 minutes download + 1 minutes of json being output in the console? Your new link does not give the AMI either, however proceeding to the launch menu we land in the link I shared previously which does show the AMI per region effortlessly, I think we can close this. Thanks.
